src/test/java/org/alliancegenome/curation_api/base/BaseITCase.java

```diff
@@ -1241,11 +1251,21 @@ public void loadGenes(List<String> modEntityIds, String taxonCurie, VocabularyTe
 	}
 
 	public void loadGOTerm(String curie, String name) throws Exception {
```

Comment:
We can get rid of the loadGOTerm methods entirely since they're doing the same thing as createGoTerm, just not returning the term (this is an issue elsewhere in the tests too, but might as well sort out this one now)